examples/cra-react15/.storybook/main.js

```diff
@@ -1,4 +1,8 @@
 module.exports = {
   stories: ['../src/stories/welcome.stories', '../src/stories/**/button.stories.js'],
-  addons: ['@storybook/addon-actions', '@storybook/addon-links'],
+  addons: [
+    '@storybook/preset-create-react-app',
```

Comment:
This starts using the new cra preset, we'll be removing the included one in the app/react in 6.0.0

The embedded preset has a problem with the css-mini-extract plugin